include/javascript/mozaik/jquery.mozaik.js

```diff
@@ -75,6 +83,34 @@ var mozaik = {
     //    return html;
     //}
 
+    widthSetInit: function() {
+        setInterval(function(){
```

Comment:
Replace setInterval with an other solution?

Why could not call this when you set the width?